src/services/schedule.js

```diff
@@ -60,16 +60,17 @@ const ScheduleService = () => {
     }
 
     // Helper function to check if a time is during the weekend
-    function duringWeekend(localTime) {
+    function duringWeekend(unixTime) {
```

Comment:
Just as a rule of thumb, let's have all our functions accept `unixTime` so it's never ambiguous.